packages/cookies/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export type { Options } from './serialize'
+export type { Cookie, CookieListItem } from './serialize'
```

Comment:
`CookieOptions` sounds more friendly to me, although I can see this is a thing defined by the spec interface